ui/src/reducers/app.js

```diff
@@ -19,6 +22,8 @@ const initialState = {
   },
   sortBy: { direction: SortByDirection.asc, index: 0 },
   isAppRequestFailed: false,
+  isSavingAppVersions: false,
```

Comment:
What do you think this should be used for? We are not doing this on the other two requests.

Reply:
It's not really needed.